settings.gradle.kts

```diff
@@ -7,18 +7,25 @@ or http://en.wikipedia.org/wiki/MIT_License
 */
 // Build file
 pluginManagement {
+    repositories {
+        gradlePluginPortal()
+        maven { setUrl("https://jcenter.bintray.com/") }
+        maven { setUrl("https://dl.bintray.com/kotlin/kotlin-eap") }
+    }
+
     resolutionStrategy {
         eachPlugin {
-            if (requested.id.id == "kotlin-platform-common") {
-                useModule("org.jetbrains.kotlin:kotlin-gradle-plugin:${requested.version}")
-            }
-
-            if (requested.id.id == "kotlin-platform-jvm") {
-                useModule("org.jetbrains.kotlin:kotlin-gradle-plugin:${requested.version}")
-            }
-
-            if (requested.id.id == "kotlin-platform-js") {
-                useModule("org.jetbrains.kotlin:kotlin-gradle-plugin:${requested.version}")
+            when (requested.id.id) {
+                "kotlin-platform-common" ->
+                    useModule("org.jetbrains.kotlin:kotlin-gradle-plugin:${requested.version}")
+                "kotlin-platform-jvm" ->
+                    useModule("org.jetbrains.kotlin:kotlin-gradle-plugin:${requested.version}")
+                "kotlin-platform-js" ->
+                    useModule("org.jetbrains.kotlin:kotlin-gradle-plugin:${requested.version}")
+                "kotlin-dce-js" ->
+                    useModule("org.jetbrains.kotlin:kotlin-gradle-plugin:${requested.version}")
```

Comment:
✏️ You could technically combine all the identical branches into one, with comma-delimited string values.

Reply:
Agreed. I'm being a bit overly verbose here and there may not be any value to it. But it _might_ be just a smidge clearer.